components/insight/SRC/org/openmicroscopy/shoola/util/PojosUtil.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2015 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *------------------------------------------------------------------------------
+ */
+
+package org.openmicroscopy.shoola.util;
+
+import omero.gateway.model.DataObject;
+import omero.gateway.model.DatasetData;
+import omero.gateway.model.PlateAcquisitionData;
+import omero.gateway.model.PlateData;
+import omero.gateway.model.ProjectData;
+import omero.gateway.model.ScreenData;
+
+/**
+ * Provides some static utility methods for dealing the with Gateway
+ * {@link DataObject}s
+ * 
+ * @author Dominik Lindner &nbsp;&nbsp;&nbsp;&nbsp; <a
+ *         href="mailto:d.lindner@dundee.ac.uk">j.burel@dundee.ac.uk</a>
+ */
+public class PojosUtil {
+
+    /**
+     * Checks if a given class is a container class, i. e. can contain multiple
+     * images
+     * 
+     * @param type
+     *            The class to check
+     * @return See above.
+     */
+    public static boolean isContainerClass(Class<?> type) {
```

Comment:
Perhaps another RFE: Is it time to start introducing marker classes into the gateway: `obj instanceof ContainerType`?